### DISCUSSION TITLE
Stato dei progetti ODS

### DISCUSSION THREAD

**Discussion by dennisangemi** [opendatasicilia/opendatasicilia.it#15] [Idee/Proposte]
Pensate possa essere importante individuare uno spazio in cui tener traccia dello **stato** dei progetti ODS (in corso, completato, morto, ecc)? Nella sezione "Cose nostre" dell'attuale sito di ODS sono presenti alcuni link che rimandano a progetti che vale la pena "resuscitare" (Esempio: http://opendatasicilia.it/mappa-opendata-ri/ ). È possibile implementare in GitHub una specie di tabella dinamica con la quale generare una visione d'insieme di tutti i progetti?

**Comment by cirospat** [comment]
grazie Dennis
condivido quanto hai scritto.
E' storia del lavoro delle persone di ODS.


Il giorno ven 9 lug 2021 alle ore 22:22 Dennis Angemi <
***@***.***> ha scritto:
